src/main/resources/entity-types/inventory/simple_holdings_records.json5

```diff
@@ -0,0 +1,740 @@
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'shelvingTitle'",
+    },
+    {
+      name: 'acquisition_format',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'acquisitionFormat'",
+    },
+    {
+      name: 'acquisition_method',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'acquisitionMethod'",
+    },
+    {
+      name: 'receipt_status',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'receiptStatus'",
+    },
+    {
+      name: 'retention_policy',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'retentionPolicy'",
+    },
+    {
+      name: 'digitization_policy',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'digitizationPolicy'",
+    },
+    {
+      name: 'copy_number',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'copyNumber'",
+    },
+    {
+      name: 'number_of_items',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'numberOfItems'",
+    },
+    {
+      name: 'receiving_history',
+      sourceAlias: 'hrd',
+      dataType: {
+        dataType: 'objectType',
+        properties: [
+          {
+            name: 'display_type',
+            dataType: {
+              dataType: 'stringType',
+            },
+            queryable: false,
+            valueGetter: "( SELECT array_agg(elems.value->>'displayType') FROM jsonb_array_elements(:sourceAlias.jsonb->'receivingHistory') AS elems)",
+            filterValueGetter: "( SELECT array_agg(lower(elems.value->>'displayType')) FROM jsonb_array_elements(:sourceAlias.jsonb->'receivingHistory') AS elems)",
+            valueFunction: 'lower(:value)',
+          },
+          {
+            name: 'entries',
+            dataType: {
+              dataType: 'arrayType',
+              itemDataType: {
+                dataType: 'objectType',
+                properties: [
```

Comment:
we have our first doubly-nested object! 🤮

Reply:
tbh I am quite interest in seeing the ref doc structure for it. :)

Reply:
https://github.com/folio-org/mod-inventory/blob/53abb78ebf6f54e8fbba85cdff7f6c18e54e3ea7/ramls/holdings-record.json#L210-L240

Reply:
oops, the outer one wasn't an array-object, just an object which should've been flattened. fixed.